edx_prefectutils/mysql.py

```diff
@@ -138,6 +181,13 @@ def _drop_temp_tables(table, connection):
         query = "CREATE TABLE {table} ({table_schema})".format(table=table + '_temp', table_schema=table_schema)
         connection.cursor().execute(query)
 
+    columns_load_order = ''
+    if load_in_order:
+        table_column_names = [name for name, __ in table_columns]
+        columns_to_load = get_columns_load_order(s3_url, table, table_column_names)
+        columns_load_order = '( {} )'.format(', '.join(columns_to_load))
```

Comment:
**Note to Reviewers:** `columns_load_order` will be added into `LOAD DATA` command below but for now I am planning to merge the changes as it is. I will check the logs to see if the current changes are working or not?